src/PrestaShopBundle/Resources/config/services/adapter/attachment.yml

```diff
@@ -60,6 +60,9 @@ services:
 
   prestashop.adapter.attachment.attachment_repository:
     class: PrestaShop\PrestaShop\Adapter\Attachment\AttachmentRepository
+    arguments:
```

Comment:
is it a BC break?

Reply:
Strictly speaking it is, but it's an acceptable one since it's:
- an adapter service (not a public API or interface)
- it should be used via Symfony DI so only those who use the class manually will have issues

So you should just park the PR as BC and write down the change so we can document it, but it's ok to change this